tasks/__init__.py

```diff
@@ -26,6 +26,12 @@
 NO_TESTS_COLLECTED = 5
 ns = Collection()
 
+try:
+    from tasks import local as local_tasks
+    ns.add_collection(Collection.from_module(local_tasks), name='local')
+except ImportError:
+    pass
```

Comment:
@Johnetordoff, do you mind adding a blurb about this to the README (it's purpose, and how to use it) so other folks will know it's there and can use it too?
```bash
cp tasks/local-dist.py tasks/local.py
docker-compose run --rm web invoke local.example_task HELLO
```